### PR TITLE
WIP: use structopt (master) 0.2

### PR DESCRIPTION
**Pull request by killercup**
Replaces #48.

cc @TeXitoi

### CODE REVIEW

File: docs/Readme.md [hunk 1]
```diff
@@ -46,15 +46,6 @@ to the `Cargo.toml` file:
 quicli = "0.1"
 ```
 
-To be able to use _all_ the features,
-also add these two goodies
-just below the "quicli" line:
-
-```toml file=Cargo.toml
-structopt = "0.1"
-serde = "1"
```
Comment (TeXitoi) [inline docs/Readme.md]: Serde case is fixée?

File: docs/commit.md [hunk 2]
```diff
@@ -20,13 +20,10 @@ authors = ["Your Name <your@email.address>"]
...
 ```
 
-As always,
-add quicli (as well as structopt and serde) as dependencies:
+Add quicli to your dependencies:
 
 ```toml file=Cargo.toml
 quicli = "0.1"
-structopt = "0.1"
-serde = "1"
```
Comment (TeXitoi) [inline docs/commit.md]: Same here

**Comment by TeXitoi** [comment]
Looks great!

File: docs/Readme.md (hunk 1 above)
Comment (killercup) [reply docs/Readme.md]: Serde isn't used in the example so it'd be weird to have it added here. (Before it made sense to say "let's always add structopt and serde too".)

**Comment by killercup** [comment]
Awesome! Let me know when 0.2 is released and I'll change this to the crates.io version and release quicli 0.2!

**Comment by TeXitoi** [comment]
You forgot to update examples

**Comment by killercup** [comment]
Good catch! Btw, I'm not sure it makes sense to use `u64` for this flag. Would you still say this is a good practice? Or should I switch to `u8`?

**Comment by TeXitoi** [comment]
u64 is the worst solution I think (after i64). Anything else is better.

Personnally, I'll opt for u8, yes.

**Comment by killercup** [comment]
Agreed. Updated.

**Comment by TeXitoi** [comment]
structopt v0.2.0 published

**Comment by killercup** [comment]
🎉 

bors r+

**Pull request merged**